content/v2.0/reference/release-notes/influxdb.md

```diff
@@ -12,6 +12,7 @@ weight: 101
 
 ### Features
 
+- Add option for Cloud users to use the `influx` CLI to interact with a Cloud instance. To download the CLI, [open the InfluxData Download page](https://portal.influxdata.com/downloads/), click **v2.0.0-beta**, and then run the command under **your OS (CLI Only)**. For information about using the `influx` CLI, see [influx - InfluxDB command line interface](/v2.0/reference/cli/influx/).
```

Comment:
actually, now that I think about it, this should probably link to get-started instructions with for instructions on placing binary in SPATH, the Catalina security issue, and influx completion steps

Reply:
Yeah, I think linking to the getting started page with instructions for downloading the binary would be better here.